easybuild/easyblocks/w/wps.py

```diff
@@ -63,17 +62,19 @@ def __init__(self, *args, **kwargs):
         self.comp_fam = None
         self.wrfdir = None
         self.compile_script = None
+        testdata_urls = ["http://www2.mmm.ucar.edu/wrf/src/data/avn_data.tar.gz"]
+        if LooseVersion(self.version) < LooseVersion('3.8'):
+            testdata_urls += ["http://www2.mmm.ucar.edu/wrf/src/wps_files/geog.tar.gz"]  # 697MB download, 16GB unpacked!
+        else:
+            testdata_urls += ["http://www2.mmm.ucar.edu/wrf/src/wps_files/geog_complete.tar.bz2"]  # 2.3GB download!
+        self.cfg['testdata'] = testdata_urls
```

Comment:
@andreas-h we should anticipate that people can potentially specify URLs in their easyconfig too, so we should check whether `self.cfg['testdata'` is stil `None` before we decide to override it...